apps/condo/pages/meter/index.tsx

```diff
@@ -594,6 +594,19 @@ interface IMeterIndexPage extends React.FC {
 
 const sortableProperties = ['date', 'clientName', 'source']
 
+function MeterPageTypeFromQuery (tabFromQuery) {
+    switch (tabFromQuery) {
+        case METER_PAGE_TYPES.meter:
+            return METER_PAGE_TYPES.meter
+        case METER_PAGE_TYPES.propertyMeter:
+            return METER_PAGE_TYPES.propertyMeter
+        case METER_PAGE_TYPES.reportingPeriod:
+            return METER_PAGE_TYPES.reportingPeriod
+        default:
+            return undefined
+    }
+}
+
```

Comment:
You can easy transform this to object. Anyway, as as far as I understand here you just check if passed type contains at `METER_PAGE_TYPES`. Why don't you just check if provided key exists at this type? You can make this done with `in` operator

Reply:
Yes, you are right. But I don't want to rewrite it. The result and performance is still the same here